sensu/resource_cluster_role_binding_test.go

```diff
@@ -0,0 +1,49 @@
+package sensu
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccResourceClusterRoleBinding_basic(t *testing.T) {
+	username := acctest.RandomWithPrefix("sensu-acctest")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccResourceClusterRoleBinding_basic(username),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"sensu_cluster_role_binding.cluster_role_binding_1", "name", "cluster_role_binding_1"),
+					resource.TestCheckResourceAttr(
+						"sensu_cluster_role_binding.cluster_role_binding_1", "users.#", "1"),
```

Comment:
rather than just testing the quantity of users, would it be reasonable to  test that we have the expected value here? (otherwise how do we know that the "right" thing is in the users list. (same goes for the groups field)